lib/internal/process/report.js

```diff
@@ -17,16 +18,29 @@ const report = {
     } else if (err === null || typeof err !== 'object') {
       throw new ERR_INVALID_ARG_TYPE('err', 'Object', err);
     }
+    const errorProperties = {};
+    const names = Object.getOwnPropertyNames(err);
+    for (let i = 0; i < names.length; i++) {
+      if (['stack', 'message'].indexOf(err[i]) !== -1)
+        errorProperties[i] = `${err[i]}`;
+    }
```

Comment:
I might be missing something, but that for loop doesn't seem right. Shouldn't it be:

```js
    for (let name of names) {
      if (!['stack', 'message'].includes(name))
        errorProperties[name] = `${err[name]}`;
    }
```

(also, maybe we want `util.inspect(err[name])` instead of `` `${err[name]}` ``?)

Reply:
I think `xxx` is a better way because inspect(xxx) will to be '"xxx"' instead of 'xxx'

Reply:
Maybe just pass through `util.inspect(err)`?

Reply:
output of `inspect(err)`:

```js
Error: foo
    at Object.<anonymous> (C:\Users\himself65\Desktop\Github\test\index.js:3:13)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
    at internal/main/run_main_module.js:17:11
```